source/Plugins/Language/Swift/SwiftArray.cpp

```diff
@@ -84,9 +84,13 @@ SwiftArrayNativeBufferHandler::SwiftArrayNativeBufferHandler(
   ProcessSP process_sp(m_exe_ctx_ref.GetProcessSP());
   if (!process_sp)
     return;
+  auto element_stride = m_elem_type.GetByteStride();
   auto opt_size = elem_type.GetByteSize(process_sp.get());
```

Comment:
Personally, I'd have written this as 
```
if (Optional<uint64_t> stride = elem_type.GetByteStride())
  m_element_stride = *stride;
```
since the type is not obvious from the name `GetByteStride`